appium/webdriver/webdriver.py

```diff
@@ -176,6 +214,16 @@ def __init__(
         By.IMAGE = MobileBy.IMAGE
         By.CUSTOM = MobileBy.CUSTOM
 
+        for extension in extensions:
+            instance = extension(self.execute)
+            if hasattr(WebDriver, instance.method_name()):
+                logger.warn(
+                    '{} is alreadt defined. The new one is overriding the old one.'.format(instance.method_name())
```

Comment:
already defined

Reply:
consider using f-strings instead

Reply:
overriding existing command might cause unexpected results. I would rather throw an exception here

Reply:
yea, I'm looking for instance wide inclusion like Ruby's `include`  to raise an error as previous add_command stuff.
`hasattr` seems class or loaded module wide